storage/storage_src.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/containers/storage/pkg/ioutils"
```

Comment:
Please move this to the other section, with other imports from outside the standard library.